src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  width: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin: 0 10px;
+  padding: 0;
+}
+
+a {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+}
+
+.nav__link {
+  text-decoration: none;
+  position: relative;
+  line-height: 23px;
+
+  padding: 18.5px 0;
```

Comment:
This padding looks weird, your can try to set padding top and bottom to 23px and remove line-height